src/pages/overview/pdf-electronic-seal-api/gettingstarted.md

```diff
@@ -207,333 +207,115 @@ public class ElectronicSeal {
-            .setVisible(sealVisible)
-            .build();
-    
-        //Set the name of TSP Provider being used.
-        providerName = "<PROVIDER_NAME>";
-    
-        //Set the access token to be used to access TSP provider hosted APIs.
-        accessToken = "<ACCESS_TOKEN>";
-    
-        //Set the credential ID.
-        credentialID = "<CREDENTIAL_ID>";
-    
-        //Set the PIN generated while creating credentials.
-        pin = "<PIN>";
-    
-        //Create CSCAuthContext instance using access token and token type.
-        cscAuthContext = new options.CSCAuthContext(accessToken, "Bearer");
-    
-        //Create CertificateCredentials instance with required certificate details.
-        certificateCredentials = options.CertificateCredentials.cscCredentialBuilder()
-            .withProviderName(providerName)
-            .withCredentialID(credentialID)
-            .withPin(pin)
-            .withCSCAuthContext(cscAuthContext)
-            .build();
-    
-        //Create SealOptions instance with sealing parameters.
-        sealOptions = new options.SealOptions.Builder(certificateCredentials, fieldOptions).build()
-    
-        //Create the PDFElectronicSealOperation instance using the SealOptions instance
-        const pdfElectronicSealOperation = pdfElectronicSeal.Operation.createNew(sealOptions);
-    
-        //Set the input source file for PDFElectronicSealOperation instance
-        pdfElectronicSealOperation.setInput(sourceFile);
-    
-        //Set the optional input seal image for PDFElectronicSealOperation instance
-        pdfElectronicSealOperation.setSealImage(sealImageFile);
-        
-        // Execute the operation and Save the result to the specified location.
-        pdfElectronicSealOperation.execute(executionContext)
-            .then(result => result.saveAsFile("output/sealedOutput.pdf"))
-            .catch(err => {
-                if(err instanceof PDFServicesSdk.Error.ServiceApiError
-                    || err instanceof PDFServicesSdk.Error.ServiceUsageError) {
-                    console.log('Exception encountered while executing operation', err);
-                } else {
-                    console.log('Exception encountered while executing operation', err);
-                }
-            });
-    
-    } catch (err) {
-    console.log('Exception encountered while executing operation', err);
-}
 ```
+
 ##### REST API
 
-```bash
+```javascript
 curl --location --request POST 'https://pdf-services.adobe.io/operation/electronicseal' \
 --header 'x-api-key: {{Placeholder for client_id}}' \
 --header 'Content-Type: application/json' \
 --header 'Authorization: Bearer {{Placeholder for token}}' \
 --data-raw '{
-    "inputDocumentAssetID": "urn:aaid:AS:UE1:23c30ee0-2c4d-xxxx-xxxx-087832fca718",
-    "sealImageAssetID": "urn:aaid:AS:UE1:23c30ee0-2e4d-xxxx-xxxx-087832fca718",
+    "inputDocumentAssetID": "urn:aaid:AS:UE1:23c30ee0-2c4d-46d6-87f2-087832fca718",
```

Comment:
these changes were fine, we shall not revert it.

Reply:
Replaced the REST API Section from eseal-ga-update